### DISCUSSION TITLE
Django multitentant types

### DISCUSSION THREAD

**Discussion by Nikkonet** [django-tenants/django-tenants#673] [Q&A]
Hello, new to the platform. Have anyone attempted the multiple types of tentants approach? I've tried following the doc and copy and pasting the files and the schema can not be found and types are not found. Can anyone help please. And thank you in advance

**Comment by tomturner** [comment]
Have you looked at the example

**Reply by Nikkonet** [reply]
Hello Mr.Turner, yes I have. Is there any way you could make a video setting up the types version of tenants?

**Comment by Nikkonet** [comment]
Would you mind shooting a tutorial video on tenant types and its set up?

**Comment by Nikkonet** [comment]
It would be greatly appreciated if you could post a short video on its
configuration. If not, I understand I will just have to seek help
elsewhere.
